src/error.ts

```diff
@@ -139,11 +139,11 @@ export class MongoError extends Error {
    *
    * @remarks
    * This class is only meant to be constructed within the driver. This constructor is
-   * not subject to semantic versioning compatiblity guarantees and may change at any time.
+   * not subject to semantic versioning compatibility guarantees and may change at any time.
    *
    * @public
    **/
-  constructor(message: string | Error, options?: { cause?: Error }) {
+  constructor(message: string, options?: { cause?: Error }) {
     super(MongoError.buildErrorMessage(message), options);
```

Comment:
`buildErrorMessage` relied on being passed the potential AggregateError, this change should result in some test failures.

Reply:
Is this comment suggesting we fix the tests or that we revert the change?

Reply:
Either, I don't feel strongly. I don't see test failures which is intriguing, maybe because the tests are still just passing in an error as the first argument?